src/components/EllipsisLoader/index.test.js

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import Enzyme, { shallow } from 'enzyme';
+import Adapter from 'enzyme-adapter-react-16';
+import EllipsisLoader from './index';
+
+Enzyme.configure({ adapter: new Adapter() });
+
+describe('Ellipsis Loader', () => {
+  it('loader__lds-ellipsis__dot should have background color #666', () => {
```

Comment:
this is only a single test, we need to add at least another one to fulfil an OKR requirement
1. we could test PropTypes or
2. we could test that it renders 4 dots

Reply:
tested option 2